e2e-tests/insurance/cypress/e2e/journeys/application-submission/multiple-policy-type/policy-conditions/submit-an-application-multiple-policy-type-not-eligible-for-small-export-builder-alternative-currency.spec.js

```diff
@@ -0,0 +1,34 @@
+import { APPLICATION } from '../../../../../../../constants';
+
+context(
+  'Insurance - submit an application - Multiple policy type - Not eligible for small export builder - Alternative currency - As an Exporter, I want to submit my completed credit insurance application, So that UKEF can process and make a decision on my application',
+  () => {
+    let referenceNumber;
+
+    before(() => {
+      cy.completeSignInAndSubmitAnApplication({
+        policyType: APPLICATION.POLICY_TYPE.MULTIPLE,
+        alternativeCurrencyPolicy: true,
+        maximumBuyerWillOwe: '5000000',
```

Comment:
could this be a constant? or something like MAXIMUM_LIMIT +1?

Reply:
I don't think it'd be wise because this is for an alternative currency.

If we use `SMALL_EXPORT_BUILDER.MAXIMUM_BUYER_WILL_OWE + 1` for example and the currency exchange rate changes, these tests could fail.

Therefore have opted for a very large amount here (5 million), which is well above `SMALL_EXPORT_BUILDER.MAXIMUM_BUYER_WILL_OWE`